examples/meta/generator/translate.py

```diff
@@ -343,7 +344,16 @@ def translateInit(self, init):
             # Optional custom SGType construction
             if typeKey == "ShogunSGType"\
                and init[0][typeKey] in self.targetDict["Init"]:
-                template = Template(self.targetDict["Init"][init[0][typeKey]])
+                origExpr = self.targetDict["Init"][init[0][typeKey]]
```

Comment:
whats the motivation here?
also for all changes in meta examples, always ping @sorig